db/migrate/20240108225031_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+      t.integer :user_id
+      t.integer :post_id
```

Comment:
👍 Good job on creating the migration file for the `likes` table! However, there's a crucial element missing in the current setup.

- [ ] Upon reviewing the database diagram, it's evident that the `likes` table should have a one-to-many relationship with the `users` table and another one-to-many relationship with the `posts` table. This association is essential for linking likes to their respective posts and users. To address this, please follow the [official documentation](https://guides.rubyonrails.org/active_record_migrations.html#creating-associations-using-references) on creating associations using references. Update your migration file accordingly to establish the relationship between the `likes` to `users` and `posts` tables using `user_id` and `post_id`. After making the necessary changes, run the migration files to ensure that the code works successfully.
![image](https://github.com/hamidazim321/blog_app/assets/55840999/158ed35c-3062-45c9-948d-6cd17624e3f0)

Keep up the good work! If you have any questions or need further guidance, feel free to ask. 🚀👍